statistics/histogram.go

```diff
@@ -959,11 +963,11 @@ func (c *Column) IsInvalid(sc *stmtctx.StatementContext, collPseudo bool) bool {
 	if collPseudo && c.NotAccurate() {
 		return true
 	}
-	if c.NDV > 0 && c.notNullCount() == 0 && sc != nil {
+	if c.Histogram.NDV > 0 && c.notNullCount() == 0 && sc != nil {
```

Comment:
Why add `.Histogram`?

Reply:
Because there is an NDV function in FMSketch. This will cause ambiguity.